content/hardware/10.mega/shields/giga-display-shield/tutorials/12.emwin-guide/content.md

```diff
@@ -0,0 +1,748 @@
+
+[emWin full demo running on the GIGA Display Shield]()
+
+## Hardware & Software Needed
+
+- [Arduino GIGA R1 WiFi](https://store.arduino.cc/products/giga-r1-wifi)
+- [Arduino GIGA Display Shield](https://store.arduino.cc/products/giga-display-shield)
+- [Arduino IDE](https://www.arduino.cc/en/software)
+
+## Downloading the Library and Core
+
+The GIGA R1 core includes the [Arduino_H7_Video](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/Arduino_H7_Video) library that handles the display.
+
+In this guide, we will be using three different libraries:
+- [Arduino_H7_Video](https://github.com/arduino/ArduinoCore-mbed/tree/main/libraries/Arduino_H7_Video), this one is bundled with the core, so make sure you have the latest version of the [Mbed core](https://github.com/arduino/ArduinoCore-mbed) installed.
+- [Arduino_GigaDisplayTouch](https://www.arduino.cc/reference/en/libraries/arduino_gigadisplaytouch/), handles touch on the GIGA Display Shield
+- [DIALOG.h](https://github.com/SEGGERMicro/emWin-Arduino-Library), the emWin library
+
+To install them, open the library manager and install the latest version by searching for **"emWin"** and **"Arduino_GigaDisplayTouch"**.
+
+In the sketch include the library like this:
+
+```arduino
+#include "DIALOG.h"
+```
+
+## emWin Library Setup
+
+In this section, we will go through the fundamental elements of an emWin sketch:
+- How to define & configure the display,
+- how to create a grid layout,
+- how to add an object to the grid,
+- how to update the display.
+
+### emWin Screen Configuration
+
+When creating elements, information about the screen and placement needs to be provided. Let's create a pointer variable that can be used whenever the screenspace needs to be used.
+
+**Full Example:**
+
+```arduino
+void setup() {
+  /* Init SEGGER emWin library. It also init display and touch controller */
+  GUI_Init();
+
+  LCD_ROTATE_SetSel(1); /* Set landscape mode */
+  WM_MULTIBUF_Enable(1); /* Enable multi buffering mode for Windows manager */
+
+  /* Create the main window. It will include all the sub-windows */
+  WM_CreateWindowAsChild(0, 0, LCD_GetXSize(), LCD_GetYSize(), WM_HBKWIN, WM_CF_SHOW, _cbWin, 0);
+}
+```
+
+### Window Manager 
+
+The examples in this tutorial will be using the window manager method. This makes it easier to manage the screen elements and widgets. However, it is also possible to create emWin sketches without the window manager, which will be shown in this section.
+
+Here is an example that will make the screens background color red and print the text "Hello world" on the screen. Lets first take a look at how this would look without the window manager:
```

Comment:
```suggestion
Here is an example that sets the screen background color to green and prints the text "Hello world" in red on the screen. Lets first take a look at how this would look without the window manager:
```